FirebaseDynamicLinks/Sources/FIRDynamicLink+Private.h

```diff
@@ -41,7 +41,7 @@ typedef NS_ENUM(NSUInteger, FIRDynamicLinkMatchConfidence) {
 
 @property(nonatomic, copy, nullable) NSString *matchMessage;
 
-@property(nonatomic, copy, readonly) NSDictionary<NSString *, id> *parametersDictionary;
```

Comment:
Can we get rid of this property from here since we are adding it in FIRDynamicLink.h so that we don't need to add readwrite?

Reply:
Hello for sure we can get rid of this property. As you said since we have moved this variable to FIRDynamicLink.h we don't need in Private header. So I've removed that and pushed.